packages/babel-parser/src/plugin-utils.js

```diff
@@ -0,0 +1,48 @@
+import type { Parser } from "./parser";
```

Comment:
Should add a `// @flow` atop this file.

Also, I don't think there's a `Parser` type export in "./parser"?

Reply:
> Should add a // @flow atop this file.

Fixing flow errors by not enabling flow is easier :rofl: 

> Also, I don't think there's a Parser type export in "./parser"?

Yeah, it should have been a default import. Classes can be imported as types, so flow strips the import.